portfolio/src/main/webapp/style.css

```
@@ -1,9 +1,116 @@
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
+
+
+body{
+  font-size: 16px;
+  color: #212121;
+  background-color: #FAFAFA;
+  font-family: Roboto;
+  margin-left: auto;
+  margin-right: auto;
+  width: fit-content;
+  max-width: 80vw;
+}
+
 #greeting-container {
   margin-top: 20px;
 }
+
+#selfie{
+    width: 200px;
+}
+
+#header{
+  display: flex;
+}
+
+#header-right{
+  padding-left: 10px;
+}
+
+#introduction{
+  margin: 0px;
+}
+
+.fullPage{
+  height: 100vh;
+}
+
+.sectionTitle{
+  font-size:24px;
+  margin: 20px;
+}
+
+.cardContainer{
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.card{
+ width: 300px; 
+ margin: 20px;
+ border-radius: 2px;
+ box-shadow:  1px 1px 5px rgba(0, 0, 0, 0.2)
+}
+
+.card > *{
+  pointer-events: none;
+}
+
+.cardImg{
+  display: block;
+  width: 300px;
+  height: 130px;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: 50% 50%;
+}
+
+.cardBottom{
+  margin: none;
+  padding: 10px;
+}
+
+.cardTitle{
+  font-size: 20px;
+  margin: 0px;
+  margin-bottom:10px
+}
+
+.cardSecondary{
+}
+
+.cardLongDesc{
+  display: none;
+}
+
+.openCard{
+  width:70vw;
```

nit: there is a space missing here.
```suggestion
  width: 70vw;
```
Generally I recommend installing a code formatter in your IDE (e.g. [prettier](https://github.com/prettier/prettier-vscode)). You can configure it to always format the code when you save and you don't have to ever think about things like this again. Also it saves you debates around how something should be formatted because you just leave it to the tool.